server/src/test/java/org/elasticsearch/discovery/MasterDisruptionIT.java

```diff
@@ -449,6 +456,56 @@ public void testVerifyApiBlocksDuringPartition() throws Exception {
 
     }
 
+    @TestLogging(
```

Comment:
really?

Reply:
would it help if I fold it into one line? :)

Reply:
can't we remove it? why is it there?

Reply:
ok maybe leave a comment